src/cli/commands/visualize.ts

```diff
@@ -0,0 +1,14 @@
+import { createCommand } from 'commander';
+
+import * as log from '../../log';
+
+import { generateVisualization } from '../../framework';
+
+export function visualize() {
+  return createCommand('visualize')
+    .description('Visualizes the collected data as a graph')
+    .action(async () => {
+      const graph = await generateVisualization();
+      log.info(`Visualize Integration SDK graph here: ${graph}`);
```

Comment:
How about dropping "Integration SDK" adjective and do just `Visualize graph here`?

Reply:
Sounds good.